file-formats/cfdf/cfdf-v1.json

```diff
@@ -0,0 +1,741 @@
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "keyUser",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP for Key Users",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP for key user extensibility",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "externalFieldName": {
+      "title": "External Field Name",
+      "description": "External field name",
+      "type": "string",
+      "maxLength": 30
+    },
+    "type": {
+      "title": "Type",
+      "description": "Custom field type",
+      "type": "object",
+      "properties": {
+        "semanticFieldType": {
+          "title": "Semantic Field Type",
+          "description": "Semantic field type",
+          "type": "string",
+          "enum": [
+            "amountWithCurrency",
+            "boolean",
+            "date",
+            "email",
+            "list",
+            "number",
+            "quantity",
+            "text",
+            "time",
+            "url",
+            "phone",
+            "amountWithoutCurrency",
+            "currencyCode",
+            "quantityOnly",
+            "unitCode",
+            "languageCode",
+            "numericalText",
+            "postingPeriod",
+            "raw",
+            "associationToStandardBo",
+            "timestamp",
+            "listBasedOnCdsView",
+            "characteristicsList",
+            "numericInterval",
+            "amountInterval",
+            "dateInterval",
+            "timeInterval",
+            "associationToCustomBo"
+          ],
+          "enumTitles": [
+            "Amount with Currency",
+            "Boolean",
+            "Date",
+            "E-Mail",
+            "List",
+            "Number",
+            "Quantity with Unit",
+            "Text",
+            "Time",
+            "Web Address",
+            "Phone Number",
+            "Amount Without Currency",
```

Comment:
This is not consistent, see above 'Amount with Currency' vs here "Amount without Currency",

Reply:
I agree, changed to "Amount without Currency" resp. Quantity without Unit